marlowe-chain-sync/libchainseek/Language/Marlowe/Runtime/ChainSync/Database/PostgreSQL.hs

```diff
@@ -354,9 +354,9 @@ performFindTxsTo credentials point = do
       )
     addressParts = Set.toList credentials >>= \case
       PaymentKeyCredential pkh ->
-        (,unPaymentKeyHash pkh) . BS.pack . pure <$> [0x00, 0x20, 0x40, 0x60]
+        (,unPaymentKeyHash pkh) . BS.pack . pure <$> [0x00, 0x20, 0x40, 0x60] <> [0x01, 0x21, 0x41, 0x61]
```

Comment:
@jhbertra, actually, I think `Credential` should include the network mask, since a credential is naturally associated with a network. Threading `NetworkId` through so many functions seems awkward.

Reply:
That sounds reasonable, unfortunately the `Cardano.Api` credentail types don't carry this information, so it breaks the mapping from one to the other. Also, it would be a major breaking change to the API because it would require the downstream components to get the networkId to call these queries (which currently only need to know the script hash or payment key hash they want transactions for).